src/ppx/graphics_util.cpp

```diff
@@ -1189,34 +1169,28 @@ Result CreateIBLTexturesFromFile(
 
     // Create irradiance texture - does not require mip maps
     std::filesystem::path irrFilePath = path.parent_path() / irrFile;
-    Result                ppxres      = CreateTextureFromFile(pQueue, irrFilePath, ppIrradianceTexture);
+    Result                ppxres;
+    {
+        ScopedTimer timer("Texture creation from file '" + irrFilePath.string() + "'");
+        ppxres = CreateTextureFromFile(pQueue, irrFilePath, ppIrradianceTexture);
+    }
     if (Failed(ppxres)) {
         return ppxres;
     }
 
-    Timer timer;
-    PPX_ASSERT_MSG(timer.Start() == ppx::TIMER_RESULT_SUCCESS, "timer start failed");
-    double fnStartTime = timer.SecondsSinceStart();
-
     // Load IBL environment map - this is stored as a bitmap on disk
     std::filesystem::path envFilePath = path.parent_path() / envFile;
     Mipmap                mipmap      = {};
-    ppxres                            = Mipmap::LoadFile(envFilePath, baseWidth, baseHeight, &mipmap, levelCount);
-    if (Failed(ppxres)) {
-        return ppxres;
+    {
+        ScopedTimer timer("Texture creation from mipmap file '" + envFilePath.string() + "'");
```

Comment:
I think this doesn't actually measure the time for CreateTextureFromMipMap call below? Which is different from the previous behaviour. Is this what we want?

Reply:
Yes right, fixing this!